tahowallet/extension
# Assign random colors to Chains in portfolio view.

Closes https://github.com/tahowallet/extension/issues/3193

Assignment is not quite random - but can be switched to truly random in the future if we deem this is _really_ something we want.

<img width="371" alt="image" src="https://user-images.githubusercontent.com/94649004/228325619-ca11fc09-1c01-4a5d-9624-b5526f45e438.png">


### To Test
- Import a wallet with assets across many networks.
- Confirm that each network is assigned a different color.

Latest build: [extension-builds-3204](https://github.com/tahowallet/extension/suites/11946270834/artifacts/626232794) (as of Fri, 31 Mar 2023 14:28:45 GMT).
